lisa/sut_orchestrator/azure/common.py

```diff
@@ -885,6 +887,33 @@ def get_storage_credential(
     return {"account_name": account_name, "account_key": key.value}
 
 
+def generate_blob_sas_token(
+    credential: Any,
+    subscription_id: str,
```

Comment:
I didn't see the method was used

Reply:
I have created a kci_kernel_source installer in internal repo. This method is used in that transformer